swarm.js

```diff
@@ -29,35 +33,42 @@ class Swarm extends EventEmitter {
       },
       close: () => this.emit('close')
     })
+
+    if (maxServerSockets !== Infinity) {
+      network.tcp.maxConnections = maxServerSockets
+      network.utp.maxConnections = maxServerSockets
```

Comment:
this means the actual server max is 2x maxServerSockets.

Reply:
oh yes..
If I'm understanding correctly it's a race (like client conns) between tcp and utp, 
so if we divide by two the maximum will distributed across - once one hits its
maximum the other will win the race every time. Is my reasoning sound?

Reply:
discussion follow up: we're going set max on both servers, and document that it's a soft limit for server sockets. we'll also add max peers, which will be a hard overall limit